styles/index.css

```diff
@@ -1,4 +1,25 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    height: 100vh;
+}
+
 body {
-    background-color: red;
+    height: 100%;
+}
+
+.header {
+    background-color: bisque;
+    margin: 0;
+    padding: 16px;
+}
+
+.header__page-title {
     color: blue;
```

Comment:
maybe this text color should be green, what do you think?

Reply:
okay thats better